src/components/smartSearch/filters/SurveySubmission/index.tsx

```diff
@@ -0,0 +1,140 @@
+            ...range,
+        });
+    };
+
+    const handleSurveySelectChange = (surveyValue: string) => {
+        if (surveyValue === DEFAULT_VALUE) {
+            setSubmittable(false);
+        }
+        else {
+            setConfig({ ...filter.config, survey: +surveyValue });
+            setSubmittable(true);
+        }
+    };
+
+    return (
+        <form onSubmit={ e => handleAddFilter(e) }>
+            <Typography style={{ lineHeight: 'unset', marginBottom: '2rem' }} variant="h4">
+                <Msg id="misc.smartSearch.survey_submission.inputString" values={{
+                    addRemoveSelect: (
+                        <StyledSelect onChange={ e => setOp(e.target.value as OPERATION) }
+                            value={ filter.op }>
+                            <MenuItem key={ OPERATION.ADD } value={ OPERATION.ADD }>
+                                <Msg id="misc.smartSearch.survey_submission.addRemoveSelect.add"/>
+                            </MenuItem>
+                            <MenuItem key={ OPERATION.SUB } value={ OPERATION.SUB }>
+                                <Msg id="misc.smartSearch.survey_submission.addRemoveSelect.sub" />
+                            </MenuItem>
+                        </StyledSelect>
+                    ),
+                    surveySelect: (
+                        <StyledSelect
+                            onChange={ e => handleSurveySelectChange(e.target.value) }
+                            SelectProps={{ renderValue: function getLabel(value) {
+                                return value === DEFAULT_VALUE ?
+                                    <Msg id="misc.smartSearch.survey_submission.surveySelect.any" /> :
+                                    <Msg
+                                        id="misc.smartSearch.survey_submission.surveySelect.survey"
+                                        values={{
+                                            surveyTitle: surveys.find(s=> s.id === value)?.title }}
+                                    />;
+                            } }}
+                            value={ filter.config.survey || DEFAULT_VALUE }>
+                            { !surveys.length && (
+                                <MenuItem key={ DEFAULT_VALUE } value={ DEFAULT_VALUE }>
+                                    <Msg id="misc.smartSearch.survey_submission.surveySelect.none" />
+                                </MenuItem>) }
+                            { surveys.map(s => (
+                                <MenuItem key={ s.id } value={ s.id }>
+                                    { s.title }
+                                </MenuItem>
+                            )) }
+                        </StyledSelect>
+                    ),
+                    timeFrame: (
+                        <TimeFrame
+                            filterConfig={{
+                                ...(filter.config.after && { after: filter.config.after }),
```

Comment:
Still not a fan of this. Is it not possible to just do `filterConfig={ filter.config }` or `filterConfig={{ after: filter.config.after, before: filter.config.before }}`?